lib/formatters/stylish-fixes.js

```diff
@@ -5,7 +5,7 @@ const fs = require('fs')
 const os = require('os')
 const path = require('path')
 const SourceCodeFixer = require('eslint/lib/util/source-code-fixer')
-const table = require('text-table')
```

Comment:
I wanted to say that we can now remove `text-table` from dependencies, but looks like we never had it as dependency? 🤔